packages/cli-repl/src/cli-repl.ts

```diff
@@ -103,46 +105,50 @@ class CliRepl {
     const version = this.buildInfo.version;
 
     this.repl = repl.start({
-      prompt: `> `,
+      prompt: '> ',
       writer: this.writer,
       completer: completer.bind(null, version),
     });
 
     const originalEval = util.promisify(this.repl.eval);
 
-    const customEval = async(input, context, filename, callback) => {
+    const customEval = async(input, context, filename, callback): Promise<any> => {
       let result;
-      let err = null;
 
       try {
         result = await this.ShellEvaluator.customEval(originalEval, input, context, filename);
       } catch (err) {
         if (isRecoverableError(input)) {
           return callback(new Recoverable(err));
-        } else {
-          result = err;
         }
+        result = err;
       }
-      callback (null, result)
+      callback(null, result);
     };
 
+    // we can assign eval, it is not a read-only property, so ignore typescript
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
Try to avoid `ts-ignore`.  Rather cast to `any` since it disables only the type checking:

```
(this.repl as any).eval = customEval;
```

Reply:
ugh typescript :/

this.repl.eval is definitely a settable property. REPL does it consistently throughout the entire codebase. bah humbug I am old and grumpy at new javascript fads